### PR TITLE
GAWB-2003 make workspace attributes actually refresh after importing them

### PR DESCRIPTION
**Pull request by helgridly**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs [hunk 1]
```diff
@@ -125,7 +125,7 @@
                                           {:header "Import Attributes" :show-cancel? true :cancel-text "Close"
                                            :content [:div {:style {:width 720 :backgroundColor "white" :padding "1em"}}
                                                       [import-data/Page (merge (select-keys props [:workspace-id])
-                                                                               {:reload (fn [] (modal/pop-modal) ((:request-refresh props)))}
+                                                                               {:on-data-imported (fn [] ((:request-refresh props)))}
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs]: `(fn [] ((:request-refresh props)))` -> `(:request-refresh props)`

**Review by MattPutnam — APPROVED** [review]
Seems legit

File: src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs (hunk 1 above)
Comment (rtitle) [reply src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs]: +1, this would work?
```
{:on-data-imported (:request-refresh props)}
```

**Comment by rtitle** [comment]
LGTM

**Review by dmohs — APPROVED** [review]

File: src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs (hunk 1 above)
Comment (helgridly) [reply src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs]: apparently so!

**Comment by helgridly** [comment]
This needs PO review. @bradtaylor can you pop by my computer sometime?

**Comment by helgridly** [comment]
PO said yes

**Pull request merged**